mmtf-serialization/src/main/java/org/rcsb/mmtf/serialization/quickmessagepackdeserialization/SB.java

```diff
@@ -0,0 +1,354 @@
+package org.rcsb.mmtf.serialization.quickmessagepackdeserialization;
+
+import java.nio.charset.Charset;
+
+
+/**
+ * Interesting thing here is that JavaScript is 3x faster than Java in handling strings.
+ * 
+ * Java StringBuilder is final, unfortunately. I guess they weren't thinking about Java2Script!
+ * 
+ * The reason we have to do this that several overloaded append methods is WAY too expensive
+ * 
+ */
+
+public class SB {

```

Comment:
Just found this one: could we get rid of it? This looks like a replacement for StringBuilder needed in jmol so that the java to js transpiler is happy. We don't need it here, let's replace it by StringBuilder